README.md

```diff
@@ -4,21 +4,108 @@
 ## What is ocdev?
 OpenShift Command line for Developers
 
+## Pre-requisites
+- OpenShift version 3.7 and up
```

Comment:
@kadel you mean v3.7.0 right? I think it is better to be more specific.

Reply:
updated